app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/initialgainorloss/initial_gain_or_loss.scala.html

```diff
@@ -45,7 +45,7 @@
         content = Some(unitInput(
             fieldId = gainKey,
             value = form.data.getOrElse(gainKey, ""),
-            label = Html(messages(s"$gainKey.label")),
+            label = Html(""),
```

Comment:
do we really not need a label anymore? Doesn't the screenshot of it have one?

Reply:
you are right. my bad :/ I've also added test cases to it.

Reply:
Done.